circle.yml

```diff
@@ -1,3 +1,21 @@
-test:
+general:
+  artifacts:
+    - "tmp"
+
+machine:
+  node:
+    version: 6.1.0
+
+dependencies:
+  post:
+    - npm install -g accesslint-cli
+
+compile:
   override:
     - bundle exec middleman build --verbose
+
+test:
+  override:
```

Comment:
Looks like `post` works here, too.

I read up a bit on it, and I think `post` is for if you want to run something _after_ your normal test suite (e.g. RSpec).

Since we are in Middleman land we don’t really have a normal test suite, but I think `post` does make sense here, since AccessLint is like a “final check.”